test/boulder-config.json

```diff
@@ -172,5 +172,5 @@
     "dnsTimeout": "10s"
   },
 
-  "subscriberAgreementURL": "http://localhost:4300/terms"
+  "subscriberAgreementURL": "http://localhost:4000/directory"
```

Comment:
localhost:4000/terms is correct here but looks like there's a bug. Filed an issue: https://github.com/letsencrypt/boulder/issues/534